pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -986,11 +986,28 @@ func updateAttestedNode(tx *gorm.DB, req *datastore.UpdateAttestedNodeRequest) (
 		return nil, sqlError.Wrap(err)
 	}
 
-	updates := AttestedNode{
-		SerialNumber:    req.CertSerialNumber,
-		ExpiresAt:       time.Unix(req.CertNotAfter, 0),
-		NewSerialNumber: req.NewCertSerialNumber,
-		NewExpiresAt:    nullableUnixTimeToDBTime(req.NewCertNotAfter),
+	m := req.InputMask
+	if m == nil {
+		m = &common.AttestedNodeMask{
```

Comment:
nit: I wonder if this is a little more maintainable not relying on an "all true" mask and instead doing an `m == nil || m.CertNotAfter` test below? In the latter case, if a field is added, we won't have to remember to set it in the "all true" mask.

Reply:
Will do!